SettingPresets/Configs/debug.yml

```diff
@@ -5,8 +5,8 @@ DEBUG_INFORMATION_FORMAT: dwarf
 ENABLE_TESTABILITY: YES
 GCC_DYNAMIC_NO_PIC: NO
 GCC_OPTIMIZATION_LEVEL: '0'
-GCC_PREPROCESSOR_DEFINITIONS: ["$(inherited)", "DEBUG=1"]
-MTL_ENABLE_DEBUG_INFO: YES
+GCC_PREPROCESSOR_DEFINITIONS: ["DEBUG=1", "$(inherited)"]
```

Comment:
Did the order of these really change? The convention has been to put `$(inherited)` first?

Reply:
So I'm not sure on this @yonaskolb ... I was just preserving the order that they were included in the plist file:

```xml
<key>GCC_PREPROCESSOR_DEFINITIONS</key>
<string>DEBUG=1 $(inherited)</string>
```

Was it right of me to interpret the plist string above as  `["DEBUG=1", "$(inherited)"]`?

Reply:
Since we won't be creating a diff when the project is opened, I'm happy to go with the standard inherited first. You made a good call though, and I don't really mind either way. If you're making another commit for the tests though, lets change it 👍